tests/test_bootstrap4/test_bootstrap.py

```diff
@@ -11,7 +11,7 @@ def test_deprecate_bootstrap_class(self):
 
         app = Flask(__name__)
         with pytest.warns(UserWarning):
-            bootstrap = Bootstrap(app)  # noqa: F841
+            bootstrap = Bootstrap(app)  # noqa: F841  # pylint:disable=unused-variable
```

Comment:
This project doesn't use Pylint, the pylint comments are useless. Please remove them.

Reply:
Done